ci/docker/serve.build.Dockerfile

```diff
@@ -4,6 +4,7 @@ ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_build
 FROM $DOCKER_IMAGE_BASE_BUILD
 
 ARG PYDANTIC_VERSION
+ARG PYTHON
```

Comment:
you can also auto detect in the script with something like:

````
PY_VERSION_CODE="$(python -c "import sys; v=sys.version_info; print(f'{v.major}.{v.minor}')")"
````

Reply:
![](https://media1.giphy.com/media/5OApoyWNC429G/giphy.gif?cid=5a38a5a2h6u34o2l7e7uf72366nr9wlzuizakjfgx5wu9srk&ep=v1_gifs_search&rid=giphy.gif&ct=g)